JoinColony/block-ingestor
# [Verified members] feat: add support for handling metadata delta motions

## Description
Added a new handler for the motion created - `ColonyMetadataDelta` event. 
The handler tries to parse the event via type guards, it currently only has support for adding verified members.
Using this approach it should be easy to implement removing of verified members.

## How to test
Easiest way is to:
1. Stop the block ingestor while running `npm run dev`, and run this branch instead.
2. spin up the CDapp on [this branch](https://github.com/JoinColony/colonyCDapp/pull/1835).
3. Choose a member that hasn't been verified yet and copy their address 
4. go to the colony dashboard home, where the temporary component for verifying members lives, paste the address and click `Verify via motion`
5. FUlly support the motion
6. run `curl -X POST --data '{"jsonrpc":"2.0","method":"evm_increaseTime","params":[300],"id":1}' localhost:8545` to forward the time by 5 minutes
7. Finalize the motion, claim rewards, ...
8. Use GraphQL query `getVerifiedMembersByColony` to check which users are added. The user you have pasted in above should show up in the `items` array

You can also check out if it works by rejecting a motion

Resolves https://github.com/JoinColony/block-ingestor/issues/162
